net-dialup/accel-ppp/files/accel-ppp-1.12.0-if_arp.patch

```diff
@@ -0,0 +1,30 @@
+# Dont use if_arp on musl libc. Causes redefinition error
+#
+# Adapted from Alpine linux patch:
```

Comment:
Was this one really adapted or taken literally?

Reply:
What do you think should it be @thesamesam

Reply:
It's fine if you really did adapt it :)

Reply:
Maybe modified would be a better word